tls/server-tls13.c

```diff
@@ -171,6 +175,40 @@ int main(int argc, char** argv)
         goto exit;
     }
 
+    /* make sure server is setup for reuse addr/port */
+    on = 1;
+    setsockopt(mSockfd, SOL_SOCKET, SO_REUSEADDR,
+            (char*)&on, (socklen_t)sizeof(on));
+#ifdef SO_REUSEPORT
```

Comment:
I don't see this defined anywhere, should it be?

Reply:
`SO_REUSEPORT` is a system type from socket.h.

Reply:
Ah...I get it...ok...so this won't trigger errors on socket impls that don't have it.